### DISCUSSION TITLE
Custom Block Data

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1110] [Help] [answered]
Hello, I want to set custom block data with for example Tags.
How can I do this?
I have experimented with some stuff, but this isn't working.
This is my code:
![image](https://user-images.githubusercontent.com/71212639/169688145-0c2d9d17-a7e5-4cf8-8b6d-ad79570fb048.png)
![image](https://user-images.githubusercontent.com/71212639/169688164-718f16bd-0318-4ae7-88e0-76fb5e9d19a7.png)
This is the error when starting the server:
![image](https://user-images.githubusercontent.com/71212639/169688204-5efd3e7f-13b4-47ad-a3f3-1c6e60cfdb87.png)
This is the error when stopping the server:
![image](https://user-images.githubusercontent.com/71212639/169688225-f8df106c-b52b-4011-83bc-3711f4c21279.png)

**Comment by TheMode** [comment, marked as answer]
Please provide code as text instead.

In the case you want to save a list of items
```java
var itemsTag = Tag.ItemStack("inventoryData").list();

block = block.withTag(itemsTag, ...);
var items = block.getTag(itemsTag);
```
